rules/manager.go

```diff
@@ -37,9 +37,9 @@ import (
 const (
 	namespace = "prometheus"
 
-	ruleTypeLabel     = "rule_type"
-	alertingRuleType  = "alerting"
-	recordingRuleType = "recording"
+	RuleTypeLabel     = "rule_type"
```

Comment:
Is the reasoning for exporting these just in case someone external might want to use them at some point? As discussed elsewhere recently, IMO we should only do that once the need arises and there's a first user.